### PR TITLE
Add service-binding slack channel

### PR DESCRIPTION
**Pull request by Avni-Sharma**
Refers https://github.com/kubernetes/community/issues/4397

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -342,6 +342,7 @@ channels:
   - name: wg-policy
   - name: wg-resource-mgmt
   - name: wg-security-audit
+  - name: wg-service-binding
```
Comment (idealhack) [inline communication/slack-config/channels.yaml]: +1 with the channel, but to use a `wg-` prefix, you need to be an "official" working group in the [sig list](https://github.com/kubernetes/community/blob/master/sig-list.md)

Comment (idealhack) [reply communication/slack-config/channels.yaml]: If you don't have the intention to create a WG, remove the prefix would be good. But if you do have that intention, either you can wait for it formed, or create without prefix now and rename it later.

Comment (Avni-Sharma) [reply communication/slack-config/channels.yaml]: Sure, I will go ahead by removing the prefix for now.

**Comment by idealhack** [comment]
/ok-to-test
/lgtm

either you can squash the commits, or someone with write access to this repo can apply the squash label :)

**Comment by cblecker** [comment]
/hold

We don’t have a service binding working group. Working groups have a specific meaning and process.

**Comment by Avni-Sharma** [comment]
> /hold
> 
> We don’t have a service binding working group. Working groups have a specific meaning and process.

Hi @cblecker I have now updated the yaml by removing the `wg` prefix, so that currently we can go ahead with `service-binding`.

**Comment by Avni-Sharma** [comment]
> > /hold
> > We don’t have a service binding working group. Working groups have a specific meaning and process.
> 
> Hi @cblecker I have now updated the yaml by removing the `wg` prefix, so that currently we can go ahead with `service-binding`.

cc @idealhack

**Comment by mrbobbytables** [comment]
I asked this in the referring GH issue, but this looks awfully close to the work being done our sig-service-catalog. Shouldn't the discussion be moved there or at least loop them into the convo?

**Comment by mrbobbytables** [comment]
There hasn't been further engagement here or in the referring issue. With that, I'm going to go ahead and close this out till an ack is had.

If this is something you'd like to move forward with, please engage with the service catalog and loop them in to the referring issue. We can then reopen or cut a new PR 👍 

/close

**Pull request closed without merging**